### PR TITLE
Update team page

### PR DESCRIPTION
**Pull request by saimn**

### CODE REVIEW

File: team.html [hunk 1]
```diff
@@ -465,7 +480,7 @@ <h3 id="core-package-contributors">Core Package Contributors<a class="paralink"
 			<li>Luigi Paioro</li>
 			<li>Luke G. Bouma</li>
 			<li>Luke Kelley</li>
-			<li>Luz Paz</li>
+			<li>luz paz</li>
```
Comment (pllim) [inline team.html:483]: Why the de-capitalization here?

Comment (saimn) [reply team.html:483]: That's the script output, it may have been fixed manually.

**Comment by dhomeier** [comment]
Wondering if we should somehow format the contributors list such as to indicate whether something is a real name or a github handle.

**Comment by pllim** [comment]
@dhomeier , that would not be easy also I don't think that is necessary. We should represent them as they want to be represented in their GitHub profile.

**Pull request merged**